src/SFA.DAS.EAS.Web.IntegrationTests/TestUtils/DataHelper/EmployerAccountsDbBuilder.cs

```diff
@@ -199,5 +187,17 @@ private static void CheckIfTaskCanceled(Task action, StringBuilder failMessage)
                     $"A DB task has been canceled, possibly because it has timed out. Timeout value is: {TestConstants.DbTimeout}");
             }
         }
+
+        public void Dispose()
+        {
+            if (!HasTransaction()) return;
+
+            _dbContext.Database.CurrentTransaction.Commit();
+        }
+
+        public bool HasTransaction()
```

Comment:
Does this look more like a property then a method?

Reply:
Agreed - Changed